Dockerfile

```diff
@@ -15,11 +17,7 @@ ENV C_INCLUDE_PATH=/usr/include/gdal
 ENV PYTHONUNBUFFERED 1
 
 COPY . /app/geoshop_back/
+RUN mv /app/geoshop_back/default_settings.py /app/geoshop_back/settings.py
 
-ARG ENV_FILE
-RUN mv ${ENV_FILE} .env && mkdir /mnt/geoshop_data
-
-RUN export $(egrep -v '^#' .env | xargs) && \
-    python manage.py migrate && \
-    python manage.py collectstatic --noinput && \
-    python manage.py compilemessages --locale=fr
+# FIXME: why was this needed?
```

Comment:
You can delete this, it was a first try to put some static files as a mount. For instance, images of metadata that are also used in other apps. But we decided to serve those files from another server.

Reply:
Alright I remove that :ok_hand: